Dockerfile

```diff
@@ -1,16 +1,18 @@
+# EXAMPLE BUILD: docker build -t birdnet-analyzer -f ./Dockerfile .
+# EXAMPLE RUN: docker run --rm --name birdnet-analyzer -v ~/Downloads:/audio birdnet-analyzer --i audio --o audio --slist audio
+
 # Build from Python 3.8 slim
 FROM python:3.8-slim
 
 # Install required packages while keeping the image small
 RUN apt-get update && apt-get install -y --no-install-recommends ffmpeg  && rm -rf /var/lib/apt/lists/*
 
 # Install required Python packages
-RUN pip3 install numpy scipy librosa 
-
-# Install Tensforflow
-RUN pip3 install tensorflow 
+RUN pip3 install numpy scipy librosa tensorflow
```

Comment:
i would recommend specifying exact versions with either a requirements.txt or a conda environment file. otherwise this will randomly break in the future

Reply:
I created a `birdnet.yml` for conda if you wanna use it:

```
name: birdnet
channels:
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=4.5=1_gnu
  - ca-certificates=2022.3.29=h06a4308_1
  - certifi=2021.10.8=py39h06a4308_2
  - ld_impl_linux-64=2.35.1=h7274673_9
  - libffi=3.3=he6710b0_2
  - libgcc-ng=9.3.0=h5101ec6_17
  - libgomp=9.3.0=h5101ec6_17
  - libstdcxx-ng=9.3.0=hd4cf53a_17
  - ncurses=6.3=h7f8727e_2
  - openssl=1.1.1n=h7f8727e_0
  - pip=21.2.4=py39h06a4308_0
  - python=3.9.12=h12debd9_0
  - readline=8.1.2=h7f8727e_1
  - setuptools=61.2.0=py39h06a4308_0
  - sqlite=3.38.2=hc218d9a_0
  - tk=8.6.11=h1ccaba5_0
  - tzdata=2022a=hda174b7_0
  - wheel=0.37.1=pyhd3eb1b0_0
  - xz=5.2.5=h7b6447c_0
  - zlib=1.2.12=h7f8727e_2
  - pip:
    - appdirs==1.4.4
    - audioread==2.1.9
    - bottle==0.12.19
    - cffi==1.15.0
    - charset-normalizer==2.0.12
    - decorator==5.1.1
    - idna==3.3
    - joblib==1.1.0
    - librosa==0.9.1
    - llvmlite==0.38.0
    - numba==0.55.1
    - numpy==1.21.6
    - packaging==21.3
    - pooch==1.6.0
    - pycparser==2.21
    - pyparsing==3.0.8
    - requests==2.27.1
    - resampy==0.2.2
    - scikit-learn==1.0.2
    - scipy==1.8.0
    - six==1.16.0
    - soundfile==0.10.3.post1
    - tflite-runtime==2.8.0
    - threadpoolctl==3.1.0
    - urllib3==1.26.9
```

Reply:
Yeah, you're right, we need to do that. Seems like it's braking already with pip not able to find a suitable TF version.